core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6043,6 +6043,25 @@ private void checkLiteral2(String expression, String expected) {
     sql.ok(expected);
   }
 
+  @Test void testSelectApproxCountDistinct() {
+    final String query = "select approx_count_distinct(\"product_id\") from \"product\"";
+    final String expectedAccurate = "SELECT COUNT(DISTINCT \"product_id\")\n"
```

Comment:
Nitpick: not a native speaker here, but I think that the opposite of "approximate" is "exact" here

Reply:
I changed the word by the way when I rebase.